styles.css

```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@500&display=swap');
```

Comment:
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600&display=swap');
```

Ти недопідключив шрифти!

![image](https://user-images.githubusercontent.com/40409129/198544950-a06f7f14-c77a-44e6-b0ad-b31aa247b45b.png)